ml-agents/mlagents/trainers/stats.py

```diff
@@ -198,8 +198,21 @@ def _maybe_create_summary_writer(self, category: str) -> None:
                 basedir=self.base_dir, category=category
             )
             os.makedirs(filewriter_dir, exist_ok=True)
+            self._delete_all_events_files(filewriter_dir)
```

Comment:
Wouldn't this be a problem when using `--resume`?

Reply:
Added a fix - the TB writer now takes in a param that tells it to not delete if `--resume` is passed.